lib/pulfalight/traject/ead2_component_config.rb

```diff
@@ -95,6 +95,13 @@
   end
 end
 
+to_field "collection_title_tesim" do |_record, accumulator, context|
+  parent = context.clipboard[:parent] || settings[:root]
```

Comment:
I understand `settings[:root]` here to reference the top of the component tree within the description 👍